src/Tickets/Commerce/Order_Modifiers/Custom_Tables/Order_Modifiers.php

```diff
@@ -107,8 +107,8 @@ protected function after_update( array $results ): array {
 
 		// Helper method to check and add indexes.
 		$results = $this->check_and_add_index( $results, 'tec_order_modifier_indx_slug', 'slug' );
-		$results = $this->check_and_add_index( $results, 'tec_order_modifier_indx_status_modifier_type_slug', 'status, modifier_type, slug' );
```

Comment:
This index was originally added to optimize coupon lookups, specifically for validating their criteria (status, modifier_type, slug).

We could always add this back in the future, I just wanted to call it out now.

Reply:
We will not add an index now that might need in the future. Indexes should be decided based on our where statements.

Especially when you add multiple columns in an index we should make sure we are using where statemets with all the indexes in the where to actually take advantage of the index.

As a result having indexed slug and modifier_type with status we are covering that. Adding the other one is just making write queries slower. Nothing more.

Reply:
I agree :100: . A lot of these indexes were introduced when we were still working on Coupons and Fees. So I'm glad your going through it again.